### DISCUSSION TITLE
Does ntime affect how a tree is grown?

### DISCUSSION THREAD

**Discussion by glyden** [kogalur/randomForestSRC#373] [Q&A] [answered]
Assuming a fixed set of hyperparameters, does the `ntime` argument have any effect on how each tree in a (survival or competing risks) random forest is grown? My impression is that it does not and that the split-statistic will use all distinct event times regardless of `ntime`. Is this correct?

I am fitting random forests for competing risks, with split-rule `logrankCR`. I only need to make predictions at a few clinically relevant timepoints, so I am planning to greatly restrict `ntime` for computational efficiency, but I want to confirm this does not adversely affect the quality of predictions. Thank you!

**Comment by ishwaran** [comment, marked as answer]
Confirmed.  The value of `ntime` does not affect the tree growing process at all.  It's meant only to improve computational performance, requiring only calculating prediction at a few values (rather than all possible event times, which can become enormous with big data).

**Reply by glyden** [reply]
Thank you for the helpful response!
